package.json

```diff
@@ -8,6 +8,7 @@
     "format:fix": "prettier --write --ignore-path .gitignore ."
   },
   "dependencies": {
+    "@fontsource/fira-code": "^4.5.13",
```

Comment:
remove in lieu of google fonts, you can just update the @​import we already have in css

Reply:
Okay going to remove.

Reply:
you'll need to run `pnpm i` again to get CI to pass as well